main.py

```diff
@@ -453,8 +456,8 @@ def populate_memes() -> None:
                 s += ", "
         s += "```"
 
-        await client.send_message(message.channel, s)
-        await client.delete_message(message)
```

Comment:
I simply kept this, but I am wondering why the original message is deleted? Also this only seems to work with the "Manage Messages" permission additionally to the ones you mentioned.

Reply:
I think the idea was to remove confirmation messages printed by the bot after a few seconds. I don't think it's worth the added complexity, so feel free to remove that feature.